media/windows/calendar-reservations/calendar-reservations.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+#
+# You need all the Python packages in requirements.txt.
+#
+# You probably want to use a Python virtual environment for this project.
+# For example:
+#
+# $ virtualenv --python=python3.8
+# $ . ./venv/bin/activate
+# $ pip3 install `cat requirements.txt`
+#
+
+import sys
+sys.path.insert(0, '../../../python')
+
+import ECC
+import Google
+import GoogleAuth
+
+import datetime
+
+from oauth2client import tools
+
+default_app_json  = 'gcalendar-reservations-client-id.json'
+default_user_json = 'user-credentials.json'
+
+verbose = True
+debug   = False
+logfile = None
+
+# dictionary of calendars that we're checking for events on
+calendars = [
+    { 
+        "name" : "Test calendar #1",
+        "id": 'c_bj96menjelb4pecracnninf45k@group.calendar.google.com',
+    },
+    {
+        "name" : "Test calendar #2",
+        "id" : 'c_ncm1ib261lp6c02i46mors4isc@group.calendar.google.com',
+    },
+    {
+        "name" : "Epiphany Events",
+        "id" : "churchofepiphany.com_9gueg54raienol399o0jtdgmpg@group.calendar.google.com",
+    }
+]
+
+#list of the domains the calendar will accept events from, will decline events from all others
+acceptable_domains = {
+    'cabral.org',
+    'epiphanycatholicchurch.org',
+    'churchofepiphany.com',
+}
+
+####################################################################
+#
+# Setup functions
+#
+####################################################################
+
+def setup_cli_args():
+    global default_app_json
+    tools.argparser.add_argument('--app-id',
+                                 default=default_app_json,
+                                 help='Filename containing Google application credentials')
+    global default_user_json
+    tools.argparser.add_argument('--user-credentials',
+                                 default=default_user_json,
+                                 help='Filename containing Google user credentials')
+
+    global verbose
+    tools.argparser.add_argument('--verbose',
+                                 action='store_true',
+                                 default=verbose,
+                                 help='If enabled, emit extra status messages during run')
+    global debug
+    tools.argparser.add_argument('--debug',
+                                 action='store_true',
+                                 default=debug,
+                                 help='If enabled, emit even more extra status messages during run')
+    global logfile
+    tools.argparser.add_argument('--logfile',
+                                 default=logfile,
+                                 help='Store verbose/debug logging to the specified file')
+
+    tools.argparser.add_argument('--donothing',
+                                 action='store_true',
+                                 help='....')
```

Comment:
Don't forget to put in a help string here (i.e., what will be in the output of `./calendar-reservations.py --help`).